recipes/rapidcheck/all/patches/0002-cmake-fix-install.patch

```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt
```

Comment:
This patch has been copied from #863 hahaha ctrl+c ctrl+v curse

Reply:
Haha, you have eagle eys! I'm sorry again! 😸